app/main.py

```diff
@@ -1,3 +1,45 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> "Distance":
```

Comment:
```suggestion
    def __add__(self, other: int) -> Distance:
```

Reply:
fix it in other cases, If you got a problem, please use this import:
`from __future__ import annotation`